spec/services/onboarding_csv_reader_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require_relative File.join(__dir__, '..', 'spec_helper')
```

Comment:
This shouldn't be necessary; having the services in the app file means they'll be autoloaded, and RSpec should be looking in the directory because of the names.  If that's not working I can do a little digging on this.